client/src/components/application/Examine/RequestInfoHeader.vue

```diff
@@ -367,10 +367,8 @@ export default {
         return  this.$store.getters.is_editing;
       },
       can_edit() {
-        // user can edit this if it is open (DRAFT, HOLD) or their INPROGRESS. If it is DRAFT or
-        // HOLD, clicking edit button will switch it to their INPROGRESS (in edit() f'n).
         if (this.is_my_current_nr) return true;
-        if (['DRAFT', 'HOLD', 'REJECTED', 'APPROVED', 'CONDITIONAL'].indexOf(this.nr_status) > -1) return true;
+        if (this.$store.getters.userHasEditRole && ['DRAFT', 'HOLD', 'REJECTED', 'APPROVED', 'CONDITIONAL', 'EXPIRED'].indexOf(this.nr_status) > -1 ) return true;
```

Comment:
EXPIRED - Outside scope of this task, and perhaps without adequate discussion. Per discussion w. Susan, removing added functionality of allowing EDIT of expired NR.